models/keep3r_network/ethereum/keep3r_network_ethereum_sources.yml

```diff
@@ -0,0 +1,16 @@
+version: 2
+
+sources:
+  - name: keep3r_network_ethereum
+    description: "decoded events and function calls for Keep3r Network on ethereum"
```

Comment:
Decoded events for [keep3r.network](https://keep3r.network/#), a marketplace for posting and accepting jobs to help run decentralized infrastructure. 

The scope of Keep3r Network is not to manage the jobs themselves, but to allow contracts to register as jobs for keepers, and keepers to register themselves as available to perform jobs. A "keeper" is a term used to refer to an external person and/or team that executes a job.